rpgpy/spcutil.py

```diff
@@ -129,3 +129,46 @@ def find_peak_edges(signal: np.array) -> Tuple[int, int]:
 
     return edge_left, edge_right
 
+
+def spectral_LDR_Galetti(spec_HV: np.array, spec_H: np.array, spec_VH_Re:np.array, spec_VH_Im: np.array,
```

Comment:
I would name this for example  `calc_spectral_ldr`. The method is good to give in the docstring but it is an implementation detail and don't belong to the function name.

There are also way too many arguments for this function. Can't you just pass `header` and `data` dictionaries from `read_rpg()`?

Reply:
I renamed the function, and changed the input variables to header and data.